docs/guide/db-dao.md

```diff
@@ -336,7 +336,7 @@ try {
     
     $transaction->commit();
     
-} catch(\Exception $e) {
+} catch(\Exception $e) { // replace \Exception with \Throwable when you are using PHP 7
```

Comment:
Shouldn't this contain example with extra `catch (\Throwable $e) {` in addition to `\Exception` as it is done inside the core code?

```php
} catch (\Exception $e) { // can be ommited when you are using PHP 7
    ...
} catch (\Throwable $e) {
   ...
}
```